pkgs/applications/networking/syncplay/default.nix

```diff
@@ -30,10 +31,11 @@ buildPythonApplication rec {
       url = "https://github.com/Syncplay/syncplay/commit/b62b038cdf58c54205987dfc52ebf228505ad03b.patch";
       hash = "sha256-pSP33Qn1I+nJBW8T1E1tSJKRh5OnZMRsbU+jr5z4u7c=";
     })
+    ./trusted_certificates.patch
```

Comment:
Might be worthwhile adding a link to https://github.com/NixOS/nixpkgs/pull/238091#issuecomment-1594820450